luna/featurevis/transformations.py

```diff
@@ -0,0 +1,152 @@
+"""
+The transformations for feature vis process
+"""
+import random
+import tensorflow as tf
+import tensorflow_addons as tfa
+
+
+
+def add_noise(img, noise, pctg):
+    """Adds noise to the image to be manipulated.
+    Args:
+        img (list): the image data to which noise should be added
+        noise (boolean): whether noise should be added at all
+        pctg (number): how much noise should be added to the image
+        channels_first (bool, optional): whether the image is encoded channels
+        first. Defaults to False.
```

Comment:
Two things:
1. there is no `channels_first` anymore.
2. can't we remove `noise` and and instead check wether `pctg` is `0`

Reply:
Percentage controls the amount of noise (or blur or rescale) and the other variable controls if the transformation should be done at all. These are inputs of the visualize_filter that enables the user to choose which transformation to make